test/helpers.lua

```diff
@@ -19,6 +20,7 @@ end
 
 local module = {
   REMOVE_THIS = {},
+  fail = busted.fail,
```

Comment:
why not `require('busted')` in the modules where it's needed?

Reply:
Not available.
Can move it to where it is needed though.